editor/src/clj/editor/fs.clj

```diff
@@ -292,12 +292,17 @@
   (cond-> (-> entry io/as-file .toPath .normalize .toAbsolutePath .toString)
           (not case-sensitive?) .toLowerCase))
 
+(def separator-char File/separatorChar)
+
 (defn below-directory?
   "Returns true if the file system entry is located below the directory."
   [^File entry ^File directory]
   (let [entry-path (comparable-path entry)
         directory-path (comparable-path directory)]
-    (and (= \/ (get entry-path (count directory-path)))
```

Comment:
On windows, it was `\`, not `/`

Reply:
Good catch. Surprised to learn this wasn't caught by the editor tests, since there are pretty extensive tests for this function. Don't we run the tests on Windows? 😬

Reply:
Yeah, we have good tests for this, but they are not run on Windows :)